omod/src/test/java/org/openmrs/module/datafilter/extension/html/ContentTest.java

```diff
@@ -41,12 +40,25 @@ public void addLocationNameLineWithMappedLocationsChecked() {
 		String expected = "<style>.listItemBoxCustom {width: 460px;padding: 2px;border: 1px solid lightgray;float: left;background-color: #EFEFEF;overflow-x: scroll;height: 200px;}</style>"
 		        + "<td valign='top'>Location</td>" + "<td valign='top'><div id='locationStrings' class='listItemBoxCustom'>"
 		        + "<span class='listItem'>"
-		        + "<input type='checkbox' name='locationStrings' id='locationStrings.L1' value='L1'><label for='locationStrings.L1'>L1</label>"
+		        + "<input type='checkbox' name='locationStrings' id=\"locationStrings.L1\" value=\"L1\"><label for=\"locationStrings.L1\">L1</label>"
 		        + "</span>" + "<span class='listItem'>"
-		        + "<input type='checkbox' name='locationStrings' id='locationStrings.L2' value='L2' checked><label for='locationStrings.L2'>L2</label>"
+		        + "<input type='checkbox' name='locationStrings' id=\"locationStrings.L2\" value=\"L2\" checked><label for=\"locationStrings.L2\">L2</label>"
 		        + "</span>" + "</div></td>";
 		assertEquals(expected, content.generate());
+	}
```

Comment:
I think you could have changed the test data for this existing test to contain an apostrophe, rather than adding a specific test case for it, no?

Reply:
Removed the test.